webui/PCWConfig.py

```diff
@@ -87,6 +87,18 @@ def get_providers_for(feature: str, namespace: str):
         return ConfigFile().getList('{}.namespace.{}/providers'.format(feature, namespace),
                                     ConfigFile().getList('{}/providers'.format(feature), ['EC2', 'AZURE', 'GCE']))
 
+    @staticmethod
+    def get_k8s_clusters_for_provider(provider: str) -> list:
+        result = []
+        providers = ConfigFile().get("k8sclusters/providers")
+        if provider in providers:
+            clusters = ConfigFile().get(f"k8sclusters/{provider}-clusters").strip()
+            for cluster in clusters.split(','):
+                cluster = cluster.strip()
+                resource_group, cluster_name = cluster.split(':')
```

Comment:
here we need some verification that string was formatted properly and if not we must throw exception here

Reply:
done